### PR TITLE
[DPM-537] don't allow play if game or casino paused

### PR DESCRIPTION
**Pull request by algys**

### CODE REVIEW

File: server/api/handlers/new_game.go [hunk 1]
```diff
@@ -54,3 +73,12 @@ func ProcessNewGameRequest(context context.Context, req *ws_interface.ApiRequest
 
 	return toGameSessionResponse(session), nil
 }
+
+func getGameListedInCasino(gameId uint64, casGames *[]*models.CasinoGame) *models.CasinoGame {
```
Comment (justefg) [inline server/api/handlers/new_game.go:77]: is pointer needed here?

Comment (algys) [reply server/api/handlers/new_game.go:77]: where exactly ? arguments or returning ?
in arguments need to avoid copying, in return value by same reasons

Comment (algys) [reply server/api/handlers/new_game.go:77]: where exactly ? arguments or returning ?
in arguments need to avoid copying, in return value by same reasons

Comment (algys) [reply server/api/handlers/new_game.go]: fixed

**Review by justefg — APPROVED** [review]

**Pull request merged**